src/main/scala/xiangshan/backend/fu/FuType.scala

```diff
@@ -65,6 +67,45 @@ object FuType extends OHEnumeration {
   val vstu = addType(name = "vstu")
 
   val intArithAll = Seq(jmp, brh, i2f, csr, alu, mul, div, fence, bku)
+  // 1 iq -> 2 seq fuType
+  def dq0OHTypeSeq(implicit p: Parameters): Seq[Seq[OHType]] = {
+    val exuParams = p(XSCoreParamsKey).backendParams.allIssueParams.take(2).map(_.exuBlockParams).flatten
+    exuParams.map(_.fuConfigs.map(_.fuType))
+  }
+  def dq1OHTypeSeq(implicit p: Parameters): Seq[Seq[OHType]] = {
+    val exuParams = p(XSCoreParamsKey).backendParams.allIssueParams.slice(2, 4).map(_.exuBlockParams).flatten
+    exuParams.map(_.fuConfigs.map(_.fuType))
+  }
```

Comment:
PLZ explain how to get these immediate numbers(2 and 4) and name them.

Reply:
Modified